src/Chapter2.hs

```diff
@@ -533,7 +545,10 @@ some information during recursive calls.
 For example, we can patch the previous function to count the number of
 steps we need to take in order to reduce the number to zero.
 
-🤔 Blitz question: can you guess what this number represents?
+🤔 Blitz question: can you guess what this number (acc) represents?
+
+The amount of time a number can be divided by 2, plus one. In other words,
+2^[acc] is the first number greater than n, in base 2.
```

Comment:
Fantastic!
Thank you for sharing the answer on the non-code question as well ❤️